doc/man3/BIO_f_prefix.pod

```diff
@@ -0,0 +1,70 @@
+=pod
+
+=head1 NAME
+
+BIO_f_prefix, BIO_set_prefix, BIO_set_indent, BIO_get_indent
+- prefix BIO filter
+
+=head1 SYNOPSIS
+
+ #include <openssl/bio.h>
+
+ const BIO_METHOD *BIO_f_prefix(void);
+ long BIO_set_prefix(BIO *b, const char *prefix);
+ long BIO_set_indent(BIO *b, long indent);
```

Comment:
Hmmm....

Ideally `indent` would be an `unsigned int` here.  I know it is a wrapper for the ctrl call so it's not really possible to change it.